File: files/CH-212.py
import pandas as pd

path = "CH-212 Remove duplicate.xlsx"

input = pd.read_excel(path, usecols="B:E", skiprows=1, nrows=15)
test = pd.read_excel(path, usecols="G", skiprows=1, nrows=9).sort_values(by="Item Code").reset_index(drop=True)

value_counts = input.melt(var_name="Column Title", value_name="Value") \
    .groupby("Value")["Column Title"].nunique() \
    .reset_index(name="Unique Column Titles") \
    .query("`Unique Column Titles` == 1") \
    .reset_index(drop=True)[["Value"]] \
    .rename(columns={"Value": "Item Code"})

print(value_counts.equals(test)) # True

File: files/CH-212.R
library(tidyverse)
library(readxl)

path = "files/CH-212 Remove duplicate.xlsx"
input = read_excel(path, range = "B2:E16")
test  = read_excel(path, range = "G2:G11") %>% arrange(`Item Code`)


value_counts = input %>%
  pivot_longer(cols = everything(), names_to = "Column Title", values_to = "Value") %>%
  summarise(n = n_distinct(`Column Title`), .by  = Value) %>%
  filter(n == 1) %>%
  select(-n) %>%
  arrange(Value)

all.equal(test$`Item Code`, value_counts$Value, check.attributes = FALSE)
#> [1] TRUE